powerauth-fido2/src/main/java/com/wultra/powerauth/fido2/rest/model/converter/AssertionChallengeConverter.java

```diff
@@ -108,16 +109,16 @@ public static AssertionChallenge convertAssertionChallengeFromOperationDetail(Op
 
         if (authenticatorDetails != null && !authenticatorDetails.isEmpty()) {
             final List<AllowCredentials> allowCredentials = new ArrayList<>();
-            boolean hasWultraModel = false;
             for (AuthenticatorDetail ad: authenticatorDetails) {
 
                 @SuppressWarnings("unchecked")
                 final List<String> transports = (List<String>) ad.getExtras().get("transports");
                 final String aaguid = (String) ad.getExtras().get("aaguid");
 
-                final byte[] credentialId = Base64.getDecoder().decode(ad.getCredentialId());
+                byte[] credentialId = Base64.getDecoder().decode(ad.getCredentialId());
```

Comment:
I would rather keep the `final` keyword and change the code below into `if-else`.